pytradfri/api/libcoap_api.py

```diff
@@ -14,15 +15,17 @@
 SERVER_ERROR_PREFIX = '5.'
 
 
-def api_factory(host, security_code):
+def api_factory(host, identity='pytradfri', psk=None, security_code=None):
     """Generate a request method."""
     def base_command(method):
 
         """Return base command."""
         return [
             'coap-client',
+            '-u',
+            identity,
             '-k',
-            security_code,
+            psk if psk else security_code,
```

Comment:
this is confusing. We should be explicit because the security code is used to get the psk and that's on a different port?